src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,86 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
+  symbol: string
+  discountPercent: string
+  isClaimed?: boolean
+  onClaim?: () => void
+}
+
+const actionsPaddingLeft = { base: 10, md: 0 }
+const columnsWidth = { base: '100%', md: '50%' }
+const columnsDirection: StackDirection = { base: 'column', md: 'row' }
+const columnsAlignItems = { md: 'center' }
+const columnsJustifyContent = { md: 'space-between' }
+const columnsSpacing = { base: 4, md: 48 }
```

Comment:
I think these should all be singular, i.e. `columnWidth`.

Reply:
Done in [2e3a203](https://github.com/shapeshift/web/pull/8605/commits/2e3a203909b4eca964e0f8520208a6f5cc7c099c)